### ISSUE TITLE
[Linux] Editor crashes when attempting to run due to qt xcb plugin

### ISSUE THREAD

**Issue by hershey5045** [o3de/o3de#2971]
**Describe the bug**
When attempting to launch editor executable via terminal, it crashes and outputs the following:
```
robin@robin-MS-7B85  ~/Repositories/o3de/build/bin/profile   development  ./Editor

qt.qpa.xcb: could not connect to display 
qt.qpa.plugin: Could not load the Qt platform plugin "xcb" in "" even though it was found.
This application failed to start because no Qt platform plugin could be initialized. Reinstalling the application may fix this problem.

Available platform plugins are: minimal, xcb.
```

**To Reproduce**
```
# clone repo
git clone https://github.com/o3de/o3de.git o3de
cd o3de
mkdir build && cd build

# generate build
cmake .. -DCMAKE_BUILD_TYPE=profile -DLY_DISABLE_TEST_MODULES=TRUE -DCMAKE_INSTALL_PREFIX=~/Repositories/o3de

# build project and run
make -j20
cd bin/profile
./Editor
```
**Expected behavior**
Editor launches successfully

**Desktop/Device (please complete the following information):**
 - Device: PC
 - OS: Ubuntu, linux kernel 5.11.0-051100-generic
 - Version 20.04.2 LTS
 - CPU AMD Ryzen 9 3900X
 - GPU AMD RX5700 XT
 - Memory 16GB

**Additional context**
Is there a simple fix for this?

**Comment by anon-oss** [comment]
I tried to install on ubuntu 20.04.2 LTS following the documentation and I am also getting this exact error. On Debian 11 also I could reproduce the same error. 
Device: OS: Linux (Ubuntu 20.04, Debian 11), cpu: Ryzen 5900, Ram: 64GB, GPU: nvidia 3060ti

**Comment by cgalvan** [comment]
Linux Editor is not yet supported, but there is a backlog task for it to be addressed here #746
